sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -754,4 +754,77 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("SPARK-33141: view should be parsed and analyzed with configs set when creating") {
+    withTable("t33141") {
```

Comment:
it's hard to read, let's just use `t`, `v1`, `v2`, etc.

Reply:
done